node_cli/operations/__init__.py

```diff
@@ -25,5 +25,6 @@
     turn_off as turn_off_op,
     turn_on as turn_on_op,
     restore as restore_op,
-    repair_sync as repair_sync_op
+    repair_sync as repair_sync_op,
+    configure_nftables as configure_firewall
```

Comment:
what is the point of exporting this function under a different name here?

it is used 6 times as `configure_nftables` and 1 time as `configure_firewall` - suggesting keeping just one version for consistency.